### PR TITLE
NETOBSERV-1190: use DNS latency instead of timestamps

### PR DESCRIPTION
**Pull request by msherif1234**
lots of dependencies that need to be merged before we get to this PR
we need https://github.com/netobserv/network-observability-operator/pull/395and its dependencies merged then
https://github.com/netobserv/network-observability-console-plugin/pull/355

### CODE REVIEW

File: web/src/components/netflow-record/record-field.tsx [hunk 1]
```diff
@@ -185,9 +158,6 @@ export const RecordField: React.FC<{
       case ColumnsId.collectiontime:
       case ColumnsId.starttime:
       case ColumnsId.endtime:
-      case ColumnsId.dnsrequesttime:
-      case ColumnsId.dnsresponsetime:
-        return dateTimeContent(typeof value === 'number' && !isNaN(value) ? new Date(value) : undefined);
```
Comment (jotak) [inline web/src/components/netflow-record/record-field.tsx:180]: This `dateTimeContent` call must be kept, for all the other date/time fields which were also using it.

File: web/src/utils/dns.ts [hunk 2]
```diff
@@ -1,21 +1,16 @@
 import { Record } from '../api/ipfix';
 
 /**
- * setRecordsDNSTimes check from provided records matching DNS IDs and set DnsRequestTimeMs / DnsResponseTimeMs
- * As the set of records may not contains all the pairs, some informations may be missing
+ * setRecordsDNSTimes check from provided records matching DNS IDs and set DnsLatencyMs
```
Comment (jotak) [inline web/src/utils/dns.ts:4]: I guess this whole function isn't necessary anymore

**Review by jotak — CHANGES_REQUESTED** [review]
Some stuff to change, but I guess that @jpinsonneau will take over this PR anyway

File: web/src/utils/dns.ts (hunk 2 above)
Comment (msherif1234) [reply web/src/utils/dns.ts]: yes removed and remove its reference

**Comment by msherif1234** [comment]
> Some stuff to change, but I guess that @jpinsonneau will take over this PR anyway

we can get this in and when @jpinsonneau comes back he will take it from here and add the additional pieces

File: web/src/utils/dns.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  { value: 3, name: 'NXDomain', description: 'Non-Existent Domain' },
+  { value: 4, name: 'NotImp', description: 'Not Implemented' },
+  { value: 5, name: 'Refused', description: 'Query Refused' },
+  { value: 6, name: 'YXDomain', description: 'Name Exists when it should not' },
+  { value: 7, name: 'YXRRSet', description: 'RR Set Exists when it should not' },
+  { value: 8, name: 'NXRRSet', description: 'RR Set that should exist does not' },
+  //{ value: 9, name: 'NotAuth', description: 'Server Not Authoritative for zone' },
+  { value: 9, name: 'NotAuth', description: 'Not Authorized' },
+  { value: 10, name: 'NotZone', description: 'Name not contained in zone' },
+  //{ value: 11, name: 'DSOTYPENI', description: 'DSO-TYPE Not Implemented' },
```
[the remark this answers is not in the document]
Comment (msherif1234) [reply web/src/utils/dns.ts:21]: https://datatracker.ietf.org/doc/html/rfc2929#section-2.3

**Pull request merged**